lib/virt_autotest/common.pm

```diff
@@ -32,7 +32,7 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             macaddress => '52:54:00:78:73:a1',
             ip => '192.168.122.106',
             distro => 'SLE_12_SP3',
-            location => 'http://mirror.suse.cz/install/SLP/SLE-12-SP3-Server-GM/x86_64/DVD1/',
+            location => 'http://schnell.suse.de/SLE12/SLE-12-SP3-Server-GM/x86_64/DVD1/',
```

Comment:
I'd be very careful. schnell.suse.de is in Nüremberg, so the whole network traffic needs to go over the NUE<->PRG VPN. mirror.suse.cz is in Prague and there was a reason why we sticked to that machine @pdostal

Reply:
We are using it only to start the installation, after that the autoyast profile will use the repos form mirror.suse.cz. 
Sending from NUE<->RPG a few hundred megabytes, don't think it will cause issue / bottleneck?

Reply:
If it isn't an issue, why switch it then?

Reply:
Regarding Felix's concern, maybe we can get some conclusion from Teodor's existing validation results? Teodor, how many jobs did you validate for your PR? It seems all are run on openqa.qam.suse.cz. Do you ever meet problems that Felix worries? If jobs number is not enough, maybe we can try more before merging the PR?

Reply:
I have run 4 jobs, but we have agreed with Felix to discuss it after he is back from vacation.

Reply:
After discussing with Felix we agree that this is not a good solution. I will close the PR